ynr/apps/utils/widgets.py

```diff
@@ -43,7 +43,7 @@ def build_attrs(self, base_attrs, extra_attrs=None):
         attrs.update(
             {
                 "inputmode": "numeric",
-                "pattern": r"[0-9\s\.,]*",
+                "pattern": r"[0-9\s\,]*",
```

Comment:
backslash (`\`) is the escape character for regex, except you don't need to escape `,` so I'm not sure if it's doing anything - this makes it confusing. It was in front of `.` because `.` is a wildcard in regex, but I'm not sure even that was necessary when enclosed in square brackets...

Reply:
I've fixed this up as suggested.